styles.css

```diff
@@ -7,16 +7,27 @@ body{
   flex-wrap: wrap;
   font-family
 }
-header, footer, aside, .main{
+header, footer, aside, aside, .main{
   padding: 1em;
+  margin: 0;
```

Comment:
This is applying `margin: 0` directly to these elements. You want to target the direct children. You would do that like so, using `header` as an example...

```css
header > * {
  margin: 0;
}
```